platform_versioned_docs/version-24.2/data_studios/index.mdx

```diff
@@ -102,6 +102,15 @@ Checkpoint files in the compute environment work directory may be shared by mult
 The cleanup process is a best effort and not guaranteed. Seqera attempts to remove the checkpoint, but it can fail if, for example, the compute environment credentials used do not have sufficient permissions to delete objects from storage buckets.
 :::
 
+## Session volume automatic resizing
+
+By default, a data studio session allocates an initial 2 GB of storage. Available disk space is continually monitored and if the available space declines below a 1 GB threshold, Data Studios dynamically resizes the file system to include an additional 2 GB of available disk space.
```

Comment:
```suggestion
By default, a session allocates an initial 2 GB of storage. Available disk space is continually monitored and if the available space drops below a 1 GB threshold, the the file system is dynamically-resized to include an additional 2 GB of available disk space.
```

Reply:
@justinegeffen You have two `the` next to each other?